test/nodes/test_retriever.py

```diff
@@ -444,6 +444,36 @@ def test_table_text_retriever_embedding(document_store, retriever, docs):
         assert isclose(doc.embedding[0], expected_value, rel_tol=0.001)
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize("retriever", ["table_text_retriever"], indirect=True)
+@pytest.mark.parametrize("document_store", ["memory"], indirect=True)
+@pytest.mark.embedding_dim(512)
+def test_table_text_retriever_embedding_only_text(document_store, retriever):
+    document_store = InMemoryDocumentStore()
+    docs = [
+        Document(content="This is a test", content_type="text"),
+        Document(content="This is another test", content_type="text"),
+    ]
+    retriever = retriever(document_store=document_store)
```

Comment:
This line needs to be removed. (The retriever is already initialized via the fixture.)